internal_transfers/actions/tests/database.spec.ts

```diff
@@ -425,7 +427,8 @@ describe("jsonFromSettlementData", () => {
       ethDelta: new Map([["0x1", bigNumber]]),
     };
     expect(jsonFromSettlementData(dummySimData)).toStrictEqual({
-      simId: "sim-id",
+      simId: simulationID,
```

Comment:
This seems like a regression to me because I can't seem to find any other place where this has changed. Considering this is already merged, might be wrong, but please just double-check.

Reply:
This is just avoiding redundancy, we have a variable `simulationID = "sim-id"` which should have been used here before.